### PR TITLE
Deploy with Travis

### PR DESCRIPTION
**Pull request by DirtyF**
Everytime we merge to master Travis should deploy the app on the gh-pages.

### CODE REVIEW

**Comment by DirtyF** [comment]
@oncletom Travis failed npm install with latest Node, what version should we stick to?

**Comment by DirtyF** [comment]
@oncletom Any clue how to fix this `npm install` failure on Travis?

**Comment by oncletom** [comment]
I'd say it is an issue with one of the dependencies and Node.js v4. [This explains why it fails to build in Node v4](https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements).

The hint is this warning in the logs:

> Starting with io.js 3 and Node.js 4, building native extensions requires C++11-compatible compiler, which seems unavailable on this VM. Please read https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements.

Eventually switch to Node v6 and it might solve the problem — or the previous link will do address it anyway.

**Comment by DirtyF** [comment]
I already tried using latest node but it doesn't work either: https://travis-ci.org/sudweb/talks/builds/155588170 

It works perfectly locally, so I don't get why it won't work on Travis with node v6.

😕

**Comment by oncletom** [comment]
Just read the link in the previous comment ;-)

**Comment by oncletom** [comment]
Now it fails because there are no tests defined. `eslint` could be added with some rules to mitigate issues but in the meantime, the `test` script could simply be replaced by `exit 0` — the build will be successful.

**Comment by DirtyF** [comment]
OK, I guess I could try to add `eslint` later, never say no to an opportunity to learn new stuff.

**Comment by oncletom** [comment]
Yeah, I would not do it right now 'cause it is more like a new feature unrelated to what you are trying to do.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+
+before_install:
+- openssl aes-256-cbc -K $encrypted_0cc7effbabbd_key -iv $encrypted_0cc7effbabbd_iv
+  -in super_secret.txt.enc -out super_secret.txt -d
```
Comment (oncletom) [inline .travis.yml]: @DirtyF Be careful, it is on a new line so the key decryption is likely to fail. Line 18+19 should fit on one single line.

**Pull request merged**